pkg/services/supportbundles/interface.go

```diff
@@ -23,10 +23,10 @@ func (s State) String() string {
 type Bundle struct {
 	UID       string `json:"uid"`
 	State     State  `json:"state"`
-	FilePath  string `json:"filePath"`
 	Creator   string `json:"creator"`
 	CreatedAt int64  `json:"createdAt"`
 	ExpiresAt int64  `json:"expiresAt"`
+	TarBytes  []byte `json:"bytesTar,omitempty"`
```

Comment:
```suggestion
	TarBytes  []byte `json:"-"`
```

Reply:
We never pass the tar with a bundle response

Reply:
but if we do that we cannot extract it from the cache/kv :sweat_smile:

As an alternative I can embed a second struct inside, wdyt?

Reply:
Oh yeah, that is right. Then we can just keep the tag I guess and make sure we are not passing it when listing.